link/link.yaml

```diff
@@ -0,0 +1,21 @@
+components:
+  schemas:
+    Link:
+      description: >-
+        Device.
+      type: object
+      properties:
+        src:
+          description: |- 
+            The unique name of a logical port of a device
+          type: string
+          x-constraint:
+          - '/components/schemas/Interface/properties/name'

```

Comment:
If there are multiple devices with identical interface names, how will we link them here? I suggest we make two fields here, one for device and another for interface

Reply:
the `interface names` are globally unique and we will get error from the client if same interface name is used in different devices

Reply:
but why would we require to have globally unique interface names? they can be unique for a device, if we use a combination of the device and the interface in the link specification.

Reply:
yes agree, added device and interface for link specification.